test/utils/async.js

```diff
@@ -1,12 +1,6 @@
 export function autoFailingAsyncTest(runner) {
```

Comment:
Maybe? As I recall you added it because people were saying that rejected promises weren't failing tests. If that is still the case then we still need the function. But I could never reproduce that problem so I can't tell if we still need this or not.